readme.md

```diff
@@ -0,0 +1,54 @@
+# Five for the Future
+
+Plugins and themes for the Five for the Future subsite: https://wordpress.org/five-for-the-future/
+
+## Contributing
+
+In order to contribute with code changes, you'll want to set up a local environment to test changes and then push the changes to a Pull Request on this Github Repository.
+
+### Initial environment setup
+
+1) Use whichever local WordPress development setup you prefer and create a new local WP site.
+2) Find the `wp-content` folder and delete it (make a backup if you have data you don't want to lose data you already have there).
+3) Fork the [five-for-the-future](https://github.com/WordPress/five-for-the-future) repository under your own Github account.
+4) Run `git clone git@github.com:WordPress/five-for-the-future.git wp-content`. Replace `WordPress/` with your Github `USERNAME/` to clone your newly forked repo instead.
```

Comment:
Maybe this, so that folks who just copy-paste don't accidentally clone the wrong repo?
```suggestion
4) Run `git clone git@github.com:[your username]/five-for-the-future.git wp-content`, replacing `[your username]` with your github username to clone your forked repo.
```